Monika After Story/game/script-apologies.rpy

```diff
@@ -231,7 +231,9 @@ label mas_apology_generic:
         $ mas_gainAffection(modifier=0.1)
         m 2tkd "What you did wasn't funny, [player]."
         m 2dkd "Please be more considerate about my feelings in the future."
-
+    elif:
```

Comment:
No `if` block for this `elif`. What exactly are you trying to accomplish with this dialogue? It's more or less identical in what is says to the `else` block above.

Reply:
Alright i'll delete it then.